libcxx/test/std/strings/basic.string/string.capacity/resize_and_overwrite.verify.cpp

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
```

Comment:
It seems better to use the new fashion.
```suggestion
// REQUIRES: std-at-least-c++23
```